openpgp/uat.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2011 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package openpgp
```

Comment:
Could you add this API in `openpgp/v2` instead, please? (And the changes in openpgp/keys.go as well.)
This package is now intended to be a drop-in replacement of x/crypto/openpgp, only.

Reply:
ok